### PR TITLE
DisplayProperties: Add options to set background color and wallpaper mode.

### PR DESCRIPTION
**Pull request by asliturk**
Wallpaper mode and background color change options are now available.

DisplayProperties

### CODE REVIEW

**Comment by supercomputer7** [comment]
I suggest to remove the dot in the end of the commit titles, and only to use dots in commit messages' body.

File: Libraries/LibGUI/ComboBox.cpp [hunk 1]
```diff
@@ -94,6 +95,15 @@ void ComboBox::set_model(NonnullRefPtr<Model> model)
     m_list_view->set_model(move(model));
 }
 
+void ComboBox::set_selected_index(size_t index)
+{
+    auto model = this->m_list_view->model();
+
+    auto resolution_index_in_model = model->index(index, 0);
```
[the remark this answers is not in the document]
Comment (asliturk) [reply Libraries/LibGUI/ComboBox.cpp]: "index" is input parameter name, so new variable name is resolution_index_in_model.

**Comment by asliturk** [comment]
> Great idea!
> 
> There is already `GUI::ColorPicker`. Let's not add another dialog for doing basically the same thing. Instead, let's improve (or replace) the existing one :)

Hi,
DisplayProperties now uses ColorPicker.
ColorChooser commit droped. I'm going to add the code to ColorPicker to improve it.  I will be create a new pool request for this change.

**Comment by awesomekling** [comment]
Fabulous :)

**Pull request merged**